site/src/pages/configuration.mdx

```diff
@@ -183,3 +183,26 @@ Example:
 The `browser` field specifies alias files exclusive to browsers. This allows you to create different bundles from your source code based on `typeof window` and `typeof document` checks - thanks to that you can, for example, remove server-only code (just for those bundles).
 
 **Note:** Those files are not meant to be consumed by browsers "as is". They just assume browser-like environment, but they still can contain for example references to `process.env.NODE_ENV` as that is meant to be replaced by a consuming bundler.
+
+### `exports`
+The exports field allows you to specify custom conditional exports for various environments. You can use this to create custom builds targeted for these specific environments. Currently, only "browser" and "worker" conditionals are supported along with their "module" and "default" variants.
+Here is an example config using "browser" and "worker" conditionals:
+
+```json
+{
+  "exports": {
+    ".": {
+      "worker": {
+        "module": "./dist/my-package.worker.esm.js",
+        "default": "./dist/my-package.worker.cjs.js"
+      },
+      "browser": {
+        "module": "./dist/my-package.browser.esm.js",
+        "default": "./dist/my-package.browser.cjs.js"
+      },
+      "module": "./dist/my-package.esm.js",
+      "default": "./dist/my-package.cjs.js"
+    }
+  }
+}
```

Comment:
As per: https://github.com/preconstruct/preconstruct/pull/435/files#r767032611

It's important to establish the shape of `package.json#exports` that we want. I propose this structure:
paths -> environments -> dev/prod

The default & module stuff etc might required to be mixed into the environments and dev/prod sections.

Reply:
Ok. I've never seen dev/prod actually used in a package. Do you have any example that I could follow?